### PR TITLE
Release plugin 1.1.1

### PR DESCRIPTION
**Pull request by rlepinski**

### CODE REVIEW

**Comment by marc-scig** [comment]
:see_no_evil: :hear_no_evil: :speak_no_evil:

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 UA Unity ChangeLog
 ==================
 
+Version 1.1.1 - October 12, 2015
+================================
+ - Fixed iOS plugin not notifying foreground pushes.
+ - Fixed Android plugin' isPushEnabled to check user notifications.
```
Comment (hcrowell) [inline CHANGELOG]: plugin's ?

**Comment by hcrowell** [comment]
:tangerine: after reviewing minor comment above ^

File: android-plugin/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java [hunk 2]
```diff
@@ -77,7 +77,7 @@ public String getIncomingPush(boolean clear) {
 
     public boolean isPushEnabled() {
         Logger.debug("UnityPlugin isPushEnabled");
-        return UAirship.shared().getPushManager().isPushEnabled();
+        return UAirship.shared().getPushManager().getUserNotificationsEnabled();
```
Comment (marc-scig) [inline android-plugin/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java]: what's up with this

**Comment by hcrowell** [comment]
:tangerine:

File: android-plugin/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java (hunk 2 above)
Comment (marc-scig) [inline android-plugin/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java]: nm, I can read

Comment (rlepinski) [inline android-plugin/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java]: Plugin was written during a time when we only had a single push flag

**Comment by marc-scig** [comment]
:hocho: :computer:

**Pull request merged**